src/DataDictionary/graph/GraphDrawer/index.js

```diff
@@ -14,23 +13,23 @@ const ReduxGraphDrawer = (() => {
     graphBoundingBox: state.ddgraph.graphBoundingBox,
     layoutInitialized: state.ddgraph.layoutInitialized,
     highlightingNode: state.ddgraph.highlightingNode,
-    highlightingNodeSVGElement: state.ddgraph.highlightingNodeSVGElement,
     relatedNodeIDs: state.ddgraph.relatedNodeIDs,
     secondHighlightingNodeCandidateIDs: state.ddgraph.secondHighlightingNodeCandidateIDs,
     pathRelatedToSecondHighlightingNode: state.ddgraph.pathRelatedToSecondHighlightingNode,
     secondHighlightingNodeID: state.ddgraph.secondHighlightingNodeID,
     isGraphView: state.ddgraph.isGraphView,
+    matchedNodeIDs: state.ddgraph.matchedNodeIDs,
+    matchedNodeIDsInNameAndDescription: state.ddgraph.matchedNodeIDsInNameAndDescription,
+    searchResult: state.ddgraph.searchResult,
+    isSearchMode: state.ddgraph.isSearchMode,
```

Comment:
does search mode need to be captured in the Redux store? it can't be captured by local state?

Reply:
`isGraphView` is controlled by the most outer component `DataDictionary`, and is shared by `DataModelStructure`, `Canvas`, `GraphDrawer`, so must be stored in Redux